src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -76,6 +79,9 @@ interface IProps {
 
     canPreview?: boolean;
     previewLoading?: boolean;
+
+    // The room to be previewed
+    // If given, the room passed is peekable
```

Comment:
I don't think this is accurate. I believe `room` is also set if we have received an invite to the room.

This will be sufficient:

```suggestion
    // A `Room` object for the room to be previewed, if we have one.
```

Reply:
yes, that is true, thanks for notice